### ISSUE TITLE
zxscanlib is no longer available in the maven repository

### ISSUE THREAD

**Issue by hrkeni** [LivotovLabs/CamView#47]
I have a legacy app that uses the zxscanlib. It's mentioned in the deprecation note that it will be available in the maven repo but since you moved to jCenter, it's not longer available.

Sorry if this is the wrong place but I can't report issues on the zxscanlib project anymore, and I felt this place is somewhat appropriate.

Thanks.

**Comment by livotov** [comment]
Yep, the old package is almost ready for jcenter. Just did not have enough time to finish it and upload. Will do in a day-two.

**Comment by GuyPaddock** [comment]
@hrkeni: See my temporary workaround here:
https://github.com/LivotovLabs/CamView/issues/39#issuecomment-206949375
